services/QuillLMS/app/assets/stylesheets/pages/privacy_and_security.scss

```diff
@@ -0,0 +1,261 @@
+.privacy-and-security-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  .divider-line {
+    width: 200px;
+    height: 1px;
+    background-color: $quill-grey-25;
+  }
+  .header-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 96px;
+    h1 {
+      color: $quill-green;
+      text-align: center;
+      font-family: "Adelle Sans";
+      font-size: 56px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 130%;
```

Comment:
Can we use one of the defined font variables for the various fonts in this file? Like `@include display-l`?